ecs-agent/daemonimages/csidriver/health/health.go

```diff
@@ -0,0 +1,68 @@
+// Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
```

Comment:
FYI https://github.com/aws/amazon-ecs-agent/blob/master/ecs-agent/csiclient/csi_client.go this client pkg will used by Agent and it's for csi client only.

Reply:
Nvm. Ignore this comment.